### DISCUSSION TITLE
How should I bind value to BoxShadow?

### DISCUSSION THREAD

**Discussion by kyuranger** [AvaloniaUI/Avalonia#16289] [Q&A] [answered]
Here is my code:
```
<Border CornerRadius="10" Background="{TemplateBinding Background}" Margin="0,0,10,10">
				<Border.BoxShadow>
					<BoxShadow Color="{TemplateBinding BoxShadowColor}" OffsetX="5" OffsetY="5" Blur="10">
					</BoxShadow>
				</Border.BoxShadow>
</Border>
```

VS reports error:

> Severity	Code	Description	Project	File	Line	Suppression State	Details
> Error (active)	AVLN:0004	Unable to find suitable setter or adder for property BoxShadow of type Avalonia.Controls:Avalonia.Controls.Border for argument Avalonia.Base:Avalonia.Media.BoxShadow, available setter parameter lists are:
> Avalonia.UnsetValueType
> Avalonia.Data.IBinding
> Avalonia.Media.BoxShadows Line 40, position 9

	
How can I bind it?

**Comment by maxkatz6** [comment, marked as answer]
BoxShadow is a struct, and you can't bind to a struct property.
Instead, you need to bind to Border.BoxShadow value. Can be done with converter too.

**Reply by kyuranger** [reply]
Thank you so much.
